x-pack/legacy/plugins/siem/public/components/ml/influencers/create_influencers.tsx

```diff
@@ -19,19 +18,14 @@ export const createKeyAndValue = (influencer: Record<string, string>): string =>
   }
 };
 
-export const createInfluencers = (score: Anomaly): JSX.Element => {
-  return (
-    <>
-      {score.influencers
-        .filter(influencer => !isEmpty(influencer))
-        .map(influencer => {
-          const keyAndValue = createKeyAndValue(influencer);
-          return (
-            <EuiFlexItem key={keyAndValue} grow={false}>
-              {keyAndValue}
-            </EuiFlexItem>
-          );
-        })}
-    </>
-  );
-};
+export const createInfluencers = (influencers: Array<Record<string, string>> = []): JSX.Element[] =>
```

Comment:
Nice clean fix with the `[]` default value here and within `getHostNameFromInfluencers` / `getNetworkFromInfluencers `. This'll still be susceptible to `null` it looks like, but shouldn't be a problem as influencers should be either `undefined` or `[]` coming back from the ML API call.